python/hail/expr/functions.py

```diff
@@ -2661,6 +2661,30 @@ def set(collection):
     return collection._method("toSet", tset(collection.dtype.element_type))
 
 
+@typecheck(t=Type)
+def empty_set(t):
+    """Returns an empty set of elements of a type `t`.
+
+    Examples
+    --------
+    .. doctest::
+
+        >>> hl.eval_expr(empty_set())
```

Comment:
how does this work?

Reply:
ah. it doesn't. You need to add these methods to the appropriate functions.rst docs.

Reply:
Good catch, fixed.